cmd/getReservation.go

```diff
@@ -34,7 +34,7 @@ func getReservation(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(token) == 0 {
-		return fmt.Errorf("no API token specified for reservation")
+		return fmt.Errorf("no API token specified for reservation. Please run the auth login command to login to TechZone.")
```

Comment:
I'd use the full command in quotes, "Please run \"itz auth login --sso\" to log in..." just so it's clear as can be to the user.

Reply:
That is a fair point! I was going back and forth on this one. I can make that change!